learn/getting_started/quick_start.md

```diff
@@ -264,7 +264,190 @@ If you have multiple indexes, you can switch between them using the indexes drop
+            </div>
+            </div>
+          `
+          }
+        })
+      ]);
+      search.start();
+  </script>
+</html>
+```
+
+The code above comes in multiple parts:
+
+- The first four lines of the `<body>` add both `searchbox` and `hits` elements. Ultimately, `instant-meilisearch` adds the search bar and search results in these elements.
+- `<script src="..">` tags are [CDNs](https://en.wikipedia.org/wiki/Content_delivery_network) that import libraries needed to run `instant-meilisearch`.
+- The JavaScript part is where you customize `instant-meilisearch`.
+
+To use `instant-meilisearch` with `npm` or `yarn` please visit [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch).
+
+:::
+
+::: tab Vue.js
+
+The following code sample uses [Vue.js](https://vuejs.org/) framework.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <link rel="stylesheet"
+    href="https://cdn.jsdelivr.net/npm/@meilisearch/instant-meilisearch/templates/basic_search.css" />
+</head>
+
+<body>
+  <div id="app" class="wrapper">
+    <ais-instant-search :search-client="searchClient" index-name="movies">
+      <ais-configure :hits-per-page.camel="10" />
+      <ais-search-box placeholder="Search here…" class="searchbox"></ais-search-box>
+      <ais-hits>
+        <div slot="item" slot-scope="{ item }">
+          <ais-highlight :hit="item" attribute="title" />
+        </div>
+      </ais-hits>
+    </ais-instant-search>
+  </div>
+</body>
+<script src="https://cdn.jsdelivr.net/npm/vue@2.6.14"></script>
+<script src="https://cdn.jsdelivr.net/npm/vue-instantsearch/vue2/umd/index.js"></script>
+<script
+  src="https://cdn.jsdelivr.net/npm/@meilisearch/instant-meilisearch/dist/instant-meilisearch.umd.min.js"></script>
+<script>
+  Vue.use(VueInstantSearch)
+  var app = new Vue({
+    el: '#app',
+    data: {
+      searchClient: instantMeiliSearch('http://127.0.0.1:7700')
+    }
+  })
+</script>
+
+</html>
+```
+
+The code above comes in multiple parts:
+
+- In `Vue.js` customization happens directly in the `<body>` tag. To make `instant-meilisearch` work with `Vue.js` some components must be added. In the above example, `ais-instant-search`, `ais-search-box` and `ais-hits` are mandatory components to generate the`instant-meilisearch` interface.
+- `<script src="..">` tags are [CDNs](https://en.wikipedia.org/wiki/Content_delivery_network) that import libraries needed to run `instant-meilisearch` with [Vue.js](https://vuejs.org).
+- The `<script>` containing JavaScript initialize `Vue.js`. The code creates a new `Vue` instance that is mandatory to link `Vue.js` with the `DOM`.
+
+To use `instant-meilisearch` in `Vue.js` using `npm` or `yarn` please visit [meilisearch-vue](https://github.com/meilisearch/meilisearch-vue).
+
+:::note
+
+The above example uses Vue 2, you can refer to [this GitHub issue](https://github.com/meilisearch/meilisearch-vue/issues/102) to learn more about Vue 3.
```

Comment:
I think it's better to inform users which version we're using a bit earlier, perhaps on this tab's first line.

For the sake of giving writing feedback and because I apparently *really* care about commas:

`The above example uses Vue 2. Refer to [this GitHub issue](https://github.com/meilisearch/meilisearch-vue/issues/102) to learn more about Vue 3.`

In my opinion that the first `,` should be a period. "The above examples uses Vue 2" is a self-sufficient statement that requires no further explanation. By adding the next clause with a comma, we are muddying that clarity. Like, a sentence that was saying one thing now has to say two things, which requires some (admittedly small) effort from the reader. I would also argue that it makes the sentence aesthetically less pleasing, because that comma makes me imagine a stressed person who does not pause long enough between sentences.